fixtures.ttar

```diff
@@ -1806,3 +2027,221 @@ Lines: 1
 extent_alloc 2 0 0 0
 Mode: 644
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Directory: fixtures/sys/vm
```

Comment:
This seems to be a duplicate of what's in `/proc/sys/vm`, is this separate directory structure needed?

Reply:
fixed